superset-frontend/src/components/ListView/ListViewStyles.less

```
@@ -104,8 +104,16 @@
     }
 
     .table-row {
+      .actions {
+        visibility: hidden;
```

let's have a little fun here ;)

```suggestion
        opacity: 0;
        transition: opacity ease-in @timing-normal;
```